rviz_ogre_vendor/CMakeLists.txt

```diff
@@ -40,8 +40,7 @@ macro(build_ogre)
     TIMEOUT 600
     CMAKE_ARGS
       -DOGRE_STATIC:BOOL=ON
-      -DOGRE_BUILD_DEPENDENCIES:BOOL=OFF
-      -DOGRE_DEPENDENCIES_DIR=${CMAKE_CURRENT_SOURCE_DIR}/ogredeps
+      -DOGRE_BUILD_DEPENDENCIES:BOOL=ON
```

Comment:
Did you try this on Linux? I think some of the built-in dependencies it builds might conflict with the system ones.

Reply:
Specifically it can cause a problem if you link ogre with something else that uses the system version of freetype, which might differ from the one ogre built.

Reply:
Maybe it's not a problem, not sure, but perhaps we should set it up so that it only builds dependencies on Windows.

Reply:
Yes, I did try it on Linux and didn't notice any problems with it.
However, I agree that it could be problematic to both have a system and a local version of the dependencies.
I'd say we keep the dependency building enabled for now and disable it if we notice any problems with it?